resources/tmdbhelper/lib/addon/consts.py

```diff
@@ -38,9 +38,9 @@
 IMAGEPATH_SMALLLOGO = 'https://image.tmdb.org/t/p/w300'
 IMAGEPATH_NEGATE = 'https://image.tmdb.org/t/p/h100_filter(negate,000,666)'
 IMAGEPATH_ALL = [IMAGEPATH_ORIGINAL, IMAGEPATH_HIGH, IMAGEPATH_LOW, IMAGEPATH_POSTER, IMAGEPATH_SMALLPOSTER, IMAGEPATH_SMALLLOGO]
-IMAGEPATH_QUALITY_POSTER = [IMAGEPATH_POSTER, IMAGEPATH_POSTER, IMAGEPATH_POSTER, IMAGEPATH_SMALLPOSTER]
+IMAGEPATH_QUALITY_POSTER = [IMAGEPATH_ORIGINAL, IMAGEPATH_POSTER, IMAGEPATH_POSTER, IMAGEPATH_SMALLPOSTER]
```

Comment:
There is no reason to get original sized posters as Kodi will simply downscale anyway with default settings
https://kodi.wiki/view/Advancedsettings.xml#imageres

At default, artwork max height for poster aspect is 720p. Anything taller is downscaled by Kodi before caching.
At w500, the IMAGEPATH_POSTER value will limit posters 500x750 (2:3 standard ratio) which is just about as close as possible to Kodi's default setting.

Reply:
Ok.